sources/packages/web/src/components/common/SearchStudents.vue

```diff
@@ -72,21 +72,26 @@
     <content-group>
       <toggle-content :toggled="!students?.length">
         <DataTable :value="students">
-          <Column field="firstName" header="First Name" :sortable="true">
+          <Column field="sin" header="SIN" :sortable="true">
+            <template #body="slotProps">
+              {{ slotProps.data.sin }}
```

Comment:
Minor but please use the `sinDisplayFormat`,

Reply:
updated